pythclient/price_feeds.py

```diff
@@ -0,0 +1,430 @@
+    "default": "base64",
+}
+
+
+class Price:
+    def __init__(self, conf, expo, price, publish_time):
+        self.conf = conf
+        self.expo = expo
+        self.price = price
+        self.publish_time = publish_time
+
+    def __str__(self):
+        return f"Price(conf={self.conf}, expo={self.expo}, price={self.price}, publish_time={self.publish_time})"
+
+    def to_dict(self):
+        return {
+            "conf": self.conf,
+            "expo": self.expo,
+            "price": self.price,
+            "publish_time": self.publish_time,
+        }
+
+    def to_json(self):
+        return json.dumps(
+            {
+                "conf": self.conf,
+                "expo": self.expo,
+                "price": self.price,
+                "publish_time": self.publish_time,
+            }
+        )
+
+
+class PriceFeed:
+    def __init__(self, ema_price, price_id, price):
+        self.ema_price = ema_price
+        self.id = price_id
+        self.price = price
+
+    def __str__(self):
+        return (
+            f"PriceFeed(ema_price={self.ema_price}, id={self.id}, price={self.price})"
+        )
+
+    def to_dict(self):
+        return {
+            "ema_price": self.ema_price.to_dict(),
+            "id": self.id,
+            "price": self.price.to_dict(),
+        }
+
+    def to_json(self):
+        return json.dumps(
+            {
+                "ema_price": self.ema_price.to_dict(),
+                "id": self.id,
+                "price": self.price.to_dict(),
+            }
+        )
+
+
+class PriceInfo:
+    def __init__(
+        self,
+        seq_num,
+        vaa,
+        publish_time,
+        attestation_time,
+        last_attested_publish_time,
+        price_feed,
+        emitter_chain_id,
+    ):
+        self.seq_num = seq_num
+        self.vaa = vaa
+        self.publish_time = publish_time
+        self.attestation_time = attestation_time
+        self.last_attested_publish_time = last_attested_publish_time
+        self.price_feed = price_feed
+        self.emitter_chain_id = emitter_chain_id
+
+    def __str__(self):
+        return (
+            f"SeqNum: {self.seq_num}\n"
+            f"VAA: {self.vaa}\n"
+            f"Publish Time: {self.publish_time}\n"
+            f"Attestation Time: {self.attestation_time}\n"
+            f"Last Attested Publish Time: {self.last_attested_publish_time}\n"
+            f"Price Feed: {self.price_feed}\n"
+            f"Emitter Chain ID: {self.emitter_chain_id}\n"
+        )
+
+    def to_dict(self):
+        return {
+            "seq_num": self.seq_num,
+            "vaa": self.vaa,
+            "publish_time": self.publish_time,
+            "attestation_time": self.attestation_time,
+            "last_attested_publish_time": self.last_attested_publish_time,
+            "price_feed": self.price_feed.to_dict(),
+            "emitter_chain_id": self.emitter_chain_id,
+        }
+
+    def to_json(self, verbose=False, target_chain=None):
```

Comment:
i think this should take vaa_format not target_chain, as the format is the real concept, whereas the target_chain is an API convenience.

(also this should be to_dict per the comment above, and i'm not convinced about target_chain in the API per the comment below)